onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -150,7 +154,9 @@ onnxruntime::ArenaExtendStrategy arena_extend_strategy = onnxruntime::ArenaExten
 #endif
 #ifdef USE_OPENVINO
 #include "core/providers/openvino/openvino_provider_factory.h"
-std::string openvino_device;
+std::string openvino_device_type;
```

Comment:
better if these are not global.
they don't need to be.

Reply:
removed enable_vpu_fast_compile and openvino_device_id have been made local. However, openvino_device_type still needs to be global to support the deprecated config setter function until it can be fully removed (https://github.com/intel/onnxruntime/blob/manohar/vpu-config/onnxruntime/python/onnxruntime_pybind_state.cc#L729)